dali/test/python/test_operator_affine_transforms.py

```diff
@@ -19,7 +19,9 @@
 import numpy as np
 import os
 
-def check_results(T1, batch_size, mat_ref, T0=None, reverse=False):
+from scipy.spatial.transform import Rotation as scipy_rotate
```

Comment:
I think we need to add scipy as an explicit test dependency in TL0_python-self-test, TL1_python-self-test_conda and TL1_python-self-test-slow.
I think librosa depends on it that is why it works now, but maybe better to be explicit about this package (just in case).

Reply:
done